### PR TITLE
Fix service loop prevention in IPVS mode.

### PR DESCRIPTION
**Pull request by fasaxc**
## Description

Remove service loop prevention rule from OUTPUT chain because
the OUTPUT chain comes _before_ DNAT when IPVS is in use.
This means that we were dropping _all_ service IP traffic.

Unconditionally drop the rule since if traffic does loop back
to us, we'll catch it on the forward path.
## Todos
- [ ] Unit tests (full coverage)
- [ ] Integration tests (delete as appropriate) In plan/Not needed/Done
- [ ] Documentation
- [ ] Backport
- [ ] Release note

## Release Note

```release-note
None required
```

### CODE REVIEW

File: rules/static.go [hunk 1]
```diff
@@ -645,13 +645,6 @@ func (r *DefaultRuleRenderer) filterOutputChain(ipVersion uint8) *Chain {
 		},
 	)
 
-	// Jump to chain for blocking service CIDR loops.
-	rules = append(rules,
-		Rule{
-			Action: JumpAction{Target: ChainCIDRBlock},
-		},
-	)
-
```
Comment (song-jiang) [inline rules/static.go:654]: Does that mean we disable service loop prevention for packet generated by local host?

Comment (fasaxc) [reply rules/static.go:654]: Yes, exactly; but if there is a loop and the packet returns to us; _then_ we'll drop it in the forward chain before we send it back out again.

**Review by song-jiang — APPROVED** [review]
LGTM

File: rules/static.go (hunk 1 above)
Comment (neiljerram) [reply rules/static.go:654]: Nice reasoning and solution!

**Review by neiljerram — APPROVED** [review]

**Pull request merged**